src/javascripts/menu.js

```diff
@@ -19,7 +19,7 @@ var about = () => {
   });
 };
 
-var template = [
+const template = [
```

Comment:
Because these changes aren't related to the ticket for toggling dark mode, I'm going to move them to a separate code quality PR.

Reply:
Ok, much thanks!